packages/datadog-plugin-aws-sdk/src/base.js

```diff
@@ -93,6 +93,7 @@ class BaseAwsSdkPlugin extends ClientPlugin {
         this.responseExtractDSMContext(operation, params, response.data ?? response, span)
       }
       this.addResponseTags(span, response)
+      this.addSpanPointer(span, response)
```

Comment:
should this be plural, i.e. `addSpanPointers`?

also, in the boto3 version, we needed both the request parameters and the response object in some cases. is that encompassed in the `response` parameter?

Reply:
oh, i see further down that the request is a possible part of the response.

still should probably be plural, tho.

Reply:
Since this is called during an S3 request, and each request only creates one span pointer, I think singular makes sense here. Although we can add multiple span pointers per Lambda execution, we only add one span pointer each time this function is called. What do you think?

Reply:
i still think it should be plural since this is a base class. the one implementation we currently have is for a single-pointer s3 requests, but we'll shortly have multi-pointer dynamodb requests. i think we should note that in the base api.

Reply:
Ah that makes sense